test/Microsoft.DotNet.Cli.Utils.Tests/project.json

```diff
@@ -1,8 +1,17 @@
 {
   "version": "1.0.0-*",
-  "compilationOptions": {
+  "buildOptions": {
     "emitEntryPoint": true,
-    "keyFile": "../../tools/Key.snk"
+    "keyFile": "../../tools/Key.snk",
+    "copyToOutput": {
+      "include": [
+        "../../TestAssets/TestProjects/OutputStandardOutputAndError/*",
```

Comment:
agreed. and you're probably right that I should have added the pack option as well, to match the previous semantic.. @ajaybhargavb - thoughts on this overall? perhaps with msbuild coming it doesn't matter much.